meta-lmp-base/recipes-bsp/u-boot/u-boot-ostree-scr-fit/boot-common.cmd.in

```diff
@@ -152,19 +152,25 @@ if test "${fiovb.is_secondary_boot}" = "0"; then
 	# That means that we've validated already boot image over secondary path
 	# So we can update stable primary path with that images
 	if test "${fiovb.bootupgrade_available}" = "1" && test "${fiovb.upgrade_available}" = "0"; then
-		echo "${fio_msg} update primary boot path with validated images ..."
+		if test -z "${custom_apply_stable}"; then
+			echo "${fio_msg} update primary boot path with validated images ..."
 
-		setenv image_path "${ostree_root}/usr/lib/firmware/${bootloader_image}"
-		setenv start_blk "${bootloader}"
-		run load_image
-		run update_primary_image
+			setenv image_path "${ostree_root}/usr/lib/firmware/${bootloader_image}"
+			setenv start_blk "${bootloader}"
+			run load_image
+			run update_primary_image
```

Comment:
Could you, please, take a look why github highlight lines 158-161, but at the same time does not highlight 163-166?

Reply:
Funny as it all seems to be just tabs.

Reply:
that's just tabs